README.md

```diff
@@ -3,3 +3,7 @@ A collection of allow list for our dependencies.
 
 > :warning: those lists are very much work-in-progress.
 > They are not related to past, current, or future internal policies at Coveo
+
+
+For more information on how Dependency Review uses this repo and how to debug issues:
+https://coveord.atlassian.net/wiki/spaces/SecChamps/pages/3442049375/GitHub+-+Dependabot+and+Dependency+Review+Supply+Chain+Compliance+Security
```

Comment:
Well, we can add this link, but this repo is Public. 😅

Reply:
ah rip